src/fusor/fusion_caller_models.py

```diff
@@ -0,0 +1,239 @@
+"""Schemas for fusion callers used in translator.py"""
```

Comment:
I'd recommend updating this description. Aren't these models the exact outputs from each fusion caller?